suite-native/ethereum-tokens/src/utils.ts

```diff
@@ -1,3 +1,9 @@
 import { NetworkSymbol } from '@suite-common/wallet-config';
+import { ethereumTokenIcons, EthereumTokenIconName } from '@trezor/icons';
+
+import { EthereumTokenSymbol } from './types';
 
 export const isEthereumAccountSymbol = (symbol: NetworkSymbol) => symbol === 'eth';
+
+export const getEthereumTokenIconName = (symbol: EthereumTokenSymbol) =>
```

Comment:
The function is not returning a `boolean` but an `IconName` string. Thus I believe that the naming is correct like it is.

Reply:
Okay, I didn't notice this. 

I think we shouldn't fallback to 'erc20' here as that's EthereumTokenIcon responsibility. Please refactor icon component to be able to accept token symbol and determine whether it has icon for this token symbol or not. If not it should fallback there as erc20 icon name. It will be slightly different API to other icons, but I think it's desirable since this is the only one where we don't have certainty that there will be icon for token.

Reply:
have a look at `isCryptoIconType` and use similar type predicate